src/rust/engine/process_execution/sandboxer/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+version = "0.0.1"
+edition = "2021"
+name = "sandboxer"
+authors = ["Pants Build <pantsbuild@gmail.com>"]
+publish = false
+
+[dependencies]
+bytes = { workspace = true }
```

Comment:
Some of these dependencies are not needed for this breakout PR, but will be used by followups, and it seemed silly to finesse that.